### PR TITLE
HIPANGULAR-27 Remove all Bootstrap files

### PR DESCRIPTION
**Pull request by patrickrobrecht**
Rewrote toolbar dropdown with Typescript;
moved Bootstrap CSS which is used into component or general styles

### CODE REVIEW

File: app/toolbar/toolbar.component.html [hunk 1]
```diff
@@ -15,48 +14,34 @@
...
-        </h2>
+        <md-icon>person</md-icon>
+        {{ 'my profile' | translate }}
       </a></li>
       <li><a (click)="logout()">
-        <h2>
-          <md-icon>logout</md-icon>
-          {{'logout' | translate}}
-        </h2>
+        <md-icon>cancel</md-icon>
```
Comment (maltehol) [inline app/toolbar/toolbar.component.html]: why did you change the icon from `logout` to `cancel`?

File: app/toolbar/toolbar.component.css [hunk 2]
```diff
@@ -73,143 +65,33 @@ md-toolbar span a.topnav {
...
-}
-
-.dropdown {
-  position: relative;
-  margin: 0;
-  height: 100%;
-}
-
-ul.dropdown-menu {
+ul.toolbar-menu {
```
Comment (maltehol) [inline app/toolbar/toolbar.component.css]: The menu should have a shadow, so the user can easily see what exactly had pop up.

Comment (patrickrobrecht) [inline app/toolbar/toolbar.component.css]: Added with latest commit

File: app/toolbar/toolbar.component.html (hunk 1 above)
Comment (patrickrobrecht) [inline app/toolbar/toolbar.component.html]: A Materical Icon `logout` doesn't exist, so I chose the one which fits best in my opinion ...

**Comment by maltehol** [comment]
The toolbar menu stays, if I click on one of the menu items.
You may use the material menu which takes care of it.
